front/lib/api/assistant/conversation.ts

```diff
@@ -872,25 +873,14 @@ export async function* postUserMessage(
     subscription,
   });
   if (isAboveMessageLimit) {
-    if (isTrial(subscription)) {
-      yield {
-        type: "user_message_error",
-        created: Date.now(),
-        error: {
-          code: "free_trial_message_limit_reached",
```

Comment:
This was not working as expected, it was displaying a notification instead of the upselling dialog.

Reply:
Should we still keep the special code and error? `test_plan_message_limit_reached` is a bit of a lie no?

Reply:
I would in favor of transitioning to a more universal naming convention, such as "limit reached." IMHO, the front-end should be responsible for guiding the upselling process.

Reply:
Fixed.